ci/gpu/build.sh

```diff
@@ -83,8 +83,8 @@ gpuci_conda_retry install -y \
                   "ucx-py=${MINOR_VERSION}"
 
 # https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
-# gpuci_conda_retry install -y "your-pkg=1.0.0"
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -y "dask>=2021.3.1"
```

Comment:
We already install dask / distributed from the `main` branch from github below so this is not needed

Reply:
```suggestion
# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
# gpuci_conda_retry install -y "your-pkg=1.0.0"
```